cclient/serialization/tests/test_json.c

```diff
@@ -481,7 +481,98 @@ void test_deserialize_attach_to_tangle(void) {
 }
 
 void test_serialize_broadcast_transactions(void) {
-  // TODO
+  serializer_t serializer;
+  const char* json_text =
+      "{\"command\":\"broadcastTransactions\",\"trytes\":["
+      "\"BYSWEAUTWXHXZ9YBZISEK9LUHWGMHXCGEVNZHRLUWQFCUSDXZHOFHWHL9MQPVJXXZLIXPX"
+      "PXF9KYEREFSKCPKYIIKPZVLHUTDFQKKVVBBN9ATTLPCNPJDWDEVIYYLGPZGCWXOBDXMLJC9V"
+      "O9QXTTBLAXTTBFUAROYEGQIVB9MJWJKXJMCUPTWAUGFZBTZCSJVRBGMYXTVBDDS9MYUJCPZ9"
+      "YDWWQNIPUAIJXXSNLKUBSCOIJPCLEFPOXFJREXQCUVUMKSDOVQGGHRNILCO9GNCLWFM9APMN"
+      "MWYASHXQAYBEXF9QRIHIBHYEJOYHRQJAOKAQ9AJJFQ9WEIWIJOTZATIBOXQLBMIJU9PCGBLV"
+      "DDVFP9CFFSXTDUXMEGOOFXWRTLFGV9XXMYWEMGQEEEDBTIJ9OJOXFAPFQXCDAXOUDMLVYRMR"
```

Comment:
can you relocate this string to an external definition/variable?

Reply:
Good idea. But only for this API?